### ISSUE TITLE
setup failed 'PosixPath' object has no attribute 'read_text'

### ISSUE THREAD

**Issue by fizwit** [aio-libs/aiohttp#3977]
## Install Issue

Same as issue #2741
File "setup.py", line 64, in <module>
        txt = (here / 'aiohttp' / '__init__.py').read_text('utf-8')
    AttributeError: 'PosixPath' object has no attribute 'read_text'

## Expected Behaviour

should install

## Actual behaviour
pip install fails 
```
pip install --prefix=/app/easybuild/software/Python/3.7.4-foss-2016b-fh1  --no-deps  --ignore-installed  --no-build-isolation  ." exited with exit code 1 and output:
Processing aiohttp-3.6.0a1
    Preparing wheel metadata: started
    Preparing wheel metadata: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /app/easybuild/software/Python/3.7.4-foss-2016b/bin/python /app/easybuild/software/Python/3.7.4-foss-2016b-fh1/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py prepare_metadata_for_build_wheel /tmp/eb-GR_GCK/tmpd9brx36v
         cwd: /tmp/eb-GR_GCK/pip-req-build-bft_cn47
    Complete output (16 lines):
    Traceback (most recent call last):
      File "/app/easybuild/software/Python/3.7.4-foss-2016b-fh1/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 207, in <module>
        main()
      File "/app/easybuild/software/Python/3.7.4-foss-2016b-fh1/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 197, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "/app/easybuild/software/Python/3.7.4-foss-2016b-fh1/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 69, in prepare_metadata_for_build_wheel
        return hook(metadata_directory, config_settings)
      File "/app/easybuild/software/Python/3.7.4-foss-2016b/lib/python3.7/site-packages/setuptools/build_meta.py", line 140, in prepare_metadata_for_build_wheel
        self.run_setup()
      File "/app/easybuild/software/Python/3.7.4-foss-2016b/lib/python3.7/site-packages/setuptools/build_meta.py", line 211, in run_setup
        self).run_setup(setup_script=setup_script)
      File "/app/easybuild/software/Python/3.7.4-foss-2016b/lib/python3.7/site-packages/setuptools/build_meta.py", line 126, in run_setup
        exec(compile(code, __file__, 'exec'), locals())
      File "setup.py", line 64, in <module>
        txt = (here / 'aiohttp' / '__init__.py').read_text('utf-8')
    AttributeError: 'PosixPath' object has no attribute 'read_text'
```

## Steps to reproduce

Tried installed differant versions 3.5.4 and 3.6.0 a1

## Your environment

 * Ubuntu 14.04
 * Python 3.7.4

**Comment by fizwit** [comment]
Solved:  Pathlib was installed on my Python 3.6 3.7 environments.

**Comment by direbutterfly** [comment]
So what was the solution? To have pathlib in only one environment?

**Comment by webknjaz** [comment]
@direbutterfly `pathlib.Path.read_text()` should available under Python 3.5+. If you don't have it, your Python installation is probably broken.
